src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -1282,12 +1282,13 @@ public void checkIfStatusDateExistsWhenItemStatusUpdated()
     assertThat(getResponse.getStatusCode(), is(HttpURLConnection.HTTP_OK));
 
     Item item = getResponse.getJson().mapTo(Item.class);
+    Instant finalStatusDate = item.getStatus().getDate().toInstant();
 
     assertThat(item.getId(), is(id.toString()));
 
     assertThat(item.getStatus().getName().value(), is("Checked out"));
 
-    assertThat(item.getStatus().getDate().toInstant(), withinSecondsBeforeNow(seconds(5)));
```

Comment:
Why would the item status date be after the current date and time? 

Even if the operations are done asynchronously, the date chosen should still be earlier than at the point of the assertion.

Have I missed something?

Reply:
That seems very odd, given that receiving the response _should_ mean that the API operation has completed and the `now` used in the assertion should be determined after that.

Thank you for investigating it.